x-pack/plugins/apm/server/lib/alerts/test_utils/index.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Logger } from 'kibana/server';
+import { Observable, of } from 'rxjs';
+import { elasticsearchServiceMock } from 'src/core/server/mocks';
+import { APMConfig } from '../../..';
+import { APMRuleRegistry } from '../../../plugin';
+
+export const createRuleTypeMocks = () => {
+  let alertExecutor: (...args: any[]) => Promise<any>;
+
+  const mockedConfig$ = of({
+    /* eslint-disable @typescript-eslint/naming-convention */
+    'apm_oss.errorIndices': 'apm-*',
+    'apm_oss.transactionIndices': 'apm-*',
+    /* eslint-enable @typescript-eslint/naming-convention */
+  }) as Observable<APMConfig>;
```

Comment:
Can you do `of<APMConfig>(` instead?

Reply:
Probably not, because the definition is incomplete. I can try though.